apps/communications/dialer/locales/shared.en-US.properties

```diff
@@ -1 +1,3 @@
 voiceMail=Voicemail
+noContactsWereFound=No contacts were found
+noContactsFoundDialogOk=ok
```

Comment:
Capital letters would be used if we want to be consistent with the other "OK" buttons.